### PR TITLE
Improve icon display

### PR DESCRIPTION
**Pull request by rugk**
The current icon is way too small and the padding was still the one used when we had the text title there...

New version:
privatebinicon

### CODE REVIEW

File: css/bootstrap/darkstrap-0.9.3.css [hunk 1]
```diff
@@ -4786,7 +4786,8 @@ tbody.collapse.in {
 
 .navbar-brand {
   float: left;
-  padding: 15px 15px;
+  padding: 3px;
+  padding-left: 8px;
```
Comment (elrido) [inline css/bootstrap/darkstrap-0.9.3.css]: This is the generated CSS from the Darkstrap project. Please move the custom CSS into css/bootstrap/privatebin.css for all Bootstrap/Darkstrap themes and check what needs to be done in css/privatebin.css for the page theme to integrate the logo there too.

Comment (rugk) [inline css/bootstrap/darkstrap-0.9.3.css]: Already did the latter. Now I also reverted the change here.

Comment (rugk) [inline css/bootstrap/darkstrap-0.9.3.css]: Ah, did not knew that.

**Comment by rugk** [comment]
This should do it now... :smile:

**Comment by elrido** [comment]
Thanks!

**Pull request merged**